hadoop-connector-examples/README.md

```diff
@@ -101,3 +101,38 @@ You can also use hadoop-connectors on Spark
 Spark (verified with Spark 2.2.1 on Ubuntu 16.04)
    spark-submit --class io.pravega.examples.spark.WordCount ${HADOOP_EXAMPLES_JAR} ${PRAVEGA_URI} ${PRAVEGA_SCOPE} ${PRAVEGA_STREAM}
 ```
```

Comment:
Please, update the `Hadoop Connector Examples` table in main `README` by adding this new sample.

Reply:
Added in the following commit